nrrd/writer.py

```diff
@@ -121,6 +121,9 @@ def write(filename, data, header={}, detached_header=False, custom_field_map=Non
         Data to save to the NRRD file
     detached_header : :obj:`bool`, optional
         Whether the header and data should be saved in separate files. Defaults to :obj:`False`
+    relative_data_path : :class:`bool`
+        whether the data file name in detached_header is saved with relative path
```

Comment:
Line wrapped at 82 characters, only wrap at 120 characters.

Capitalize W in whether. 

detached_header should be detached header because you are not referencing the boolean variable but rather the actual detached header.

I think we should include a note that this parameter is ignored if the data is attached just to be safe.

And, you say that the "data file name is saved with a relative path", but that sounds odd. The data filename isn't what is saved as a relative path, it's the data file itself.

Altogether, here's what I'm thinking. Comments welcome
```
Whether the data in a detached header is saved as a relative path or absolute path. This parameter is ignored if the data file is attached to the header file. Defaults to :obj:`True`
```

Reply:
I don't think that is odd, rather the suggested one sounds like because data is not saved in the detached header. However, counter suggestion combining with yours:
> Whether the data file name in detached header is saved with a relative path or absolute path. 
        This parameter is ignored if there is no detached header. Defaults to :obj:`True`

Reply:
PR is ready, let me know your final thoughts on this.